__tests__/database/migration/file/index.test.ts

```diff
@@ -7,6 +7,6 @@ describe('TESTING all migration file', () => {
     // Execution
 
     // Assertion
-    expect(allMigrations.length).toEqual(16);
+    expect(allMigrations.length).toEqual(17);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case has been updated correctly to reflect the addition of a new migration version. The expected length of `allMigrations` is now 17, which matches the number of migrations after the new one has been added.
```diff
-    expect(allMigrations.length).toEqual(16);
+    expect(allMigrations.length).toEqual(17);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->